packages/office-ui-fabric-react/src/components/List/examples/List.Ghosting.Example.tsx

```diff
@@ -2,16 +2,26 @@ import * as React from 'react';
 import { FocusZone, FocusZoneDirection } from 'office-ui-fabric-react/lib/FocusZone';
 import { List } from 'office-ui-fabric-react/lib/List';
 import { Image, ImageFit } from 'office-ui-fabric-react/lib/Image';
-import { mergeStyleSets, getTheme, DefaultFontStyles, FontSizes, getFocusStyle } from 'office-ui-fabric-react/lib/Styling';
+import { ITheme, mergeStyleSets, getTheme, DefaultFontStyles, FontSizes, getFocusStyle } from 'office-ui-fabric-react/lib/Styling';
 
 export type IExampleItem = { name: string; thumbnail: string };
 
 export interface IListGhostingExampleProps {
   items: IExampleItem[];
 }
 
-const theme = getTheme();
-const classNames = mergeStyleSets({
+export interface IListGhostingExampleClassObject {
```

Comment:
```suggestion
interface IListGhostingExampleClassObject {
```

We don't need to export this interface since it is scoped to the example.